src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_LINE_NAME = "supply";
+    private static final String BUY_LINE_NAME = "buy";
+    private static final String RESULT_LINE_NAME = "result";
+    private static final String DELIMITER = ",";
+    private int supplyTotal;
+    private int buyTotal;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        supplyTotal = 0;
+        buyTotal = 0;
+        readFile(fromFileName);
+        writeToFile(createReport(), toFileName);
+    }
+
+    private void readFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                calculateTotal(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file" + fromFileName, e);
+        }
+    }
+
+    private void calculateTotal(String line) {
+        String amount;
+        if (line.contains(SUPPLY_LINE_NAME)) {
```

Comment:
better use split(",") method and compare the exact line with equals

Reply:
why?